src/game/hsfdraw.c

```diff
@@ -2755,9 +2755,9 @@ static s32 MakeCalcNBT(HsfObject *arg0, HsfFace *arg1, s16 arg2, s16 arg3) {
     NBTB.z = temp_r31[temp_r25].z - temp_r31[temp_r24].z;
     VECNormalize(&NBTB, &NBTB);
     VECCrossProduct(&NBTB, &sp10, &NBTT);
-    GXPosition3s16(sp10.x * 256.0f, sp10.y * 256.0f, sp10.z * 256.0f);
-    GXPosition3s16(NBTB.x * 256.0f, NBTB.y * 256.0f, NBTB.z * 256.0f);
-    GXPosition3s16(NBTT.x * 256.0f, NBTT.y * 256.0f, NBTT.z * 256.0f);
+    GXNormal3s16(sp10.x * 256.0f, sp10.y * 256.0f, sp10.z * 256.0f);
+    GXNormal3s16(NBTB.x * 256.0f, NBTB.y * 256.0f, NBTB.z * 256.0f);
+    GXNormal3s16(NBTT.x * 256.0f, NBTT.y * 256.0f, NBTT.z * 256.0f);
```

Comment:
We don't have a `GXNormal3s16` defined in the project, so these calls are breaking functions `MakeCalcNBT` and `MakeNBT`. This issue wasn't caught by our checks because hsfdraw is not linked into the project yet.

Could you add `GXNormal3s16` to GXVert.h as part of your PR? The implementation should be the same as `GXPosition3s16`.

Reply:
Done!